### ISSUE TITLE
Wikidata duplicate Lexeme

### ISSUE THREAD

**Issue by vladarta** [nyurik/lexicator#3]
Hi

I found this
https://www.wikidata.org/wiki/Lexeme:L145256
and
https://www.wikidata.org/wiki/Lexeme:L145257
seems to be the same

May be we need some system to indicate that one russian wiktionary page was already uploaded to wikidata by nyurik/lexicator for other users (or may be also you) not to reload existing data.

**Comment by Infovarius** [comment]
These are not duplicates. These are homonyms (with different etymology and attestation). Unmerged.

**Comment by whysage** [comment]
Thanks for answer!

As for me only Senses parts are different, so why not to combine all other in one and add all senses?
Or I missed something?

**Comment by Infovarius** [comment]
Etymology is different too

**Comment by whysage** [comment]
Please explain what you mean by etymology in context of lexeme?
I do not see this part in page structure (only Statements, Senses and Forms exists)

**Comment by nyurik** [comment]
My understanding is that some words could have the same "base lexeme", but wouldn't have the same etymology, and thus might not have the same forms.  I could be wrong.

**Comment by whysage** [comment]
Yes, I agree that if 2 words have different forms 2 records for lexeme must be created
but in my example
https://www.wikidata.org/wiki/Lexeme:L145256
and
https://www.wikidata.org/wiki/Lexeme:L145257

all forms are exact the same so think it's no sense to create 2 records and all data in Lexeme:L145256 and Lexeme:L145257 can be merged to one record

**Comment by nyurik** [comment]
@whysage I'm reading https://ru.wiktionary.org/wiki/%D0%BD%D0%B0%D1%87%D0%B0%D0%BB -- it seems the composition of the words is different. The first one breaks up as

Приставка: на-; корень: -ча-; суффикс: -л.

and the second one has no suffix, and the last "-л" is a part of the stem:

Приставка: на-; корень: -чал-.

The word composition is what determines a single vs multiple lexemes in this case because it is set on the whole lexeme, not on the lexeme's forms.

P.S. note that at the moment we don't store the word's composition, only the stem (which is identical in both cases), but i think when we were discussing such cases, we decided it was harder to split them up later on than to create multiple ones, one per wiktionary section.

**Comment by whysage** [comment]
@nyurik thank you!
Now I see the difference between these lexemes and it makes sense to create both of them.

**Comment by Infovarius** [comment]
you may also compare French Lexemes for 'tour': L2330, L2331 and L2332. They were widely discussed as examples of separation.

**Comment by whysage** [comment]
@Infovarius thank you!
